spec/subnets-lifecycle.md

```diff
@@ -0,0 +1,68 @@
+# Subnet Lifecycle
+A subnet is a new subsystem that a user can spawn from a parent subnet in a permissionless and on-demand way, depending on scalability requirements. Subnets have separate consensus algorithms and cryptoeconomic rules from their parent subnet. Subnets are firewalled from the parent network.
+
+Subnets begin with a chosen "rootnet". In case of L2 subnets, “rootnets” refer to a layer 1 blockchain, such as Filecoin or Ethereum. Child subnets are spawned from the rootnet and the rootnet becomes the parent subnet.
+
+Each subnet can have any number of child subnets, while each child subnet only has one parent subnet. Subnets can scale infinitely, to layer 2 and beyond. A single hierarchy tree begins at the chosen rootnet.
+
+Subnets within a single hierarchy tree have native communication protocols and are able to transfer assets and state without a custom bridge.
+
+## **Lifecycle**
+
+The lifecycle of a subnet begins when it’s deployed and ends when the subnet is closed.
+
+Before a subnet is created, one must specify the validator power allocation mode, which is called the `PermissionMode`. There are three kinds of permission mode at the moment:
+
+- `Collateral`: This means the power of the validator comes from the collateral staked. New validators can `join` the subnet, also `stake` more collateral, `unstake` collateral and finally `leave` the subnet
+- `Federated`: The power of the validator is set by the owner of the subnet
+- `Static`: The power of the validator is set when the subnet is created. This mode is used mainly for debugging and testing
+
+At the time of subnet creation, a minimum validator count requirement is set by the subnet creator.  If the subnet’s permission mode is collateral, the creator could also set the minimum collateral requirement for the subnet to be bootstrapped. A standard fee for the transaction on the parent network will be paid for the transaction that establishes the subnet.
+
+Before the minimal requirements are met, the subnet is in a `preBootstrap` state, once those conditions are met, the subnet is in a `postBootstap` state. Any operations performed on the subnet in `preBootstrap` state is recorded in the subnet `genesis`.
+
+For normal users, they can perform:
```

Comment:
What is a normal user? Is it a Validator that's not an Owner, or just a 3rd party who has a wallet in the subnet?

Reply:
3rd party who has a wallet in the subnet. I will rephrase this a little.